get-latest-version.py

```diff
@@ -51,6 +51,15 @@
 github_token = os.environ['GITHUB_TOKEN']
 github_repo = os.environ['GITHUB_REPOSITORY']
 github = Github(github_token)
+repo = github.get_repo(github_repo)
+
+# Method for checking if GitHub Actions has already opened an issue with this title
+def github_issue_exists(title):
+    open_issues = repo.get_issues(state='open')
```

Comment:
```suggestion
    open_issues = repo.get_issues(state='open', creator='github-actions[bot]')
```
that way you can only search for the title afterwards

Reply:
That doesn't seem to be a valid argument to `get_issues` in the version of `pygithub` we have :confused:

Reply:
Ah, indeed, the documentation doesn't specify the availability but it's only available since 1.35 and we have 1.26